core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-shell/dot-experiments-shell-routing.module.ts

```diff
@@ -29,6 +29,13 @@ const routes: Routes = [
                             '../dot-experiments-configuration/dot-experiments-configuration.module'
                         )
                     ).DotExperimentsConfigurationModule
+            },
+            {
+                path: 'reports',
+                title: 'Experiment Reports',
```

Comment:
the tittles in this routes need `internationalization` ?

Reply:
@fmontes? I did this `TitleStrategy` (`core-web/apps/dotcms-ui/src/app/shared/services/dot-title-strategy.service.ts`), where we can inject the pipe (not sure) and translate it direct like the directive of `DotDropdown`

Reply:
translated.